packages/navitem/src/react/context.tsx

```diff
@@ -18,6 +18,7 @@ export interface ContextValue {
   selected: boolean
   // eslint-disable-next-line camelcase
   UNSAFE_stylesFor?: Partial<Record<AllowedSelectors, CSSProperties>>
+  rest?: Record<string, unknown>
```

Comment:
why a named prop for rest?

Reply:
It's not a prop.  It's on context.  Context is the way that we're pass stuff to inner elements now.

Reply:
ah...got it 👍